neo4j-quickstart/src/test/java/org/acme/neo4j/Neo4jResource.java

```diff
@@ -0,0 +1,34 @@
+package org.acme.neo4j;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.testcontainers.containers.Neo4jContainer;
+
+public class Neo4jResource implements QuarkusTestResourceLifecycleManager {
+
+    private final Neo4jContainer NEO4J = new Neo4jContainer<>("neo4j:4.0.0").withAdminPassword(null)
+        .withExposedPorts(7687)
+        .withCreateContainerCmdModifier(cmd -> cmd
+            .withHostName("localhost")
+            .getHostConfig().withPortBindings(new PortBinding(Ports.Binding.bindPort(7687), new ExposedPort(7687))));
```

Comment:
The interest of test container combined with this approach is that you don't need to set up the exposed port. It will pick a random available port.

Reply:
I had a big problem to make it work with the native test, and found that solution. But I just tried it again without that and it worked, so I must have made a mistake...

Reply:
@mcserra it used to be the case, however look at how the other QS uses the QuarkusTestResource, and then it works :-)

Reply:
Its pretty cool :) Is it ready then?